.github/workflows/build-distros.yml

```diff
@@ -2,7 +2,7 @@
 # It is aimed at checking new commits don't introduce any breaking build changes.
 name: build-distros
 
-on: [pull_request]
+on: [push]
```

Comment:
Was this just for testing?

Reply:
this was based on the discussion about keeping this the same as CircleCI until we migrate and get everyone on board that we are changing how we trigger workflows.

Reply:
Ah, I thought we were just going to do it straight out of the gate.

Reply:
I'll post something asking the question so its a fast follow, but I think it would be good to get full awareness before we change this.